### PR TITLE
Add documentation for "present" and "filled" validation rules

### PR DESCRIPTION
**Pull request by themsaid**
Added documentation reference to the new `present` validation rule as well as the `filled` validation rule.

### CODE REVIEW

File: validation.md [hunk 1]
```diff
@@ -701,6 +703,16 @@ The field under validation must be numeric.
 The field under validation must match the given regular expression.
 
 **Note:** When using the `regex` pattern, it may be necessary to specify rules in an array instead of using pipe delimiters, especially if the regular expression contains a pipe character.
+
+<a name="rule-present"></a>
+#### Present
```
Comment (danieliancu) [inline validation.md]: Should be lowercase.

Comment (themsaid) [inline validation.md]: Fixed. Thanks for the heads up.

**Comment by taylorotwell** [comment]
This rule listing is in alphabetical order.

**Comment by themsaid** [comment]
Sorted alphabetically.

**Pull request merged**